pkgs/by-name/fl/flottbot/version.patch

```diff
@@ -0,0 +1,37 @@
+diff --git a/version/version.go b/version/version.go
+index 9ca8aba..d9bd968 100644
+--- a/version/version.go
++++ b/version/version.go
+@@ -4,9 +4,6 @@ package version
+ 
+ import (
+ 	"fmt"
+-	"runtime/debug"
+-
+-	"github.com/Masterminds/semver/v3"
+ )
+ 
+ // Version supplies the semantic version.
+@@ -14,20 +11,8 @@ var Version string
```

Comment:
how about replacing this patch with 

`ldflags=" -X github.com/flattbot/version.Version=${version}`  ?

Reply:
I don't have strong opinion, I am just offering options.

patch is more correct, ldflags is less code :-)